src/status_im/transport/handlers.cljs

```diff
@@ -18,16 +18,27 @@
             [status-im.transport.message.v1.group-chat :as v1.group-chat]
             [status-im.data-store.transport :as transport-store]))
 
+(defn receive-message [cofx chat-id message]
+  (let [{:keys [payload sig]} message
+        status-message (-> payload
+                           transport.utils/to-utf8
+                           transit/deserialize)]
+    (when (and sig status-message)
+      (message/receive status-message (or chat-id sig) sig cofx))))
+
+(defn receive-whisper-message [{:keys [db] :as cofx} [js-error js-message chat-id]]
+  (reduce (fn [fx message]
```

Comment:
docstring explaining that a whisper message can contain a vector of messages or a single message

Reply:
i will make sure that whisper message will always be a single now, at least if you are speaking about `message` parameter, that's always single message. No need for doc exactly here

Reply:
maybe my comment should be one line above, I mean that js-message is a vector of message. is it always the case ?